src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -27,11 +24,11 @@ class JobFactory(WMObject):
     def __init__(self,
                  package='WMCore.DataStructs',
                  subscription=None,
-                 generators=[],
+                 generators=None,
                  limit = 0):
         self.package = package
         self.subscription  = subscription
-        self.generators    = generators
+        self.generators    = generators if generators else []
```

Comment:
Besides shutting up pylint, it makes no sense. Why don't we just default to [] since lambda function is called on it later on?